src/secp256k1.c

```diff
@@ -706,6 +706,38 @@ static int secp256k1_ec_commit_verify(const secp256k1_context* ctx, const secp25
     return secp256k1_gej_is_infinity(&pj);
 }
 
+static uint64_t s2c_opening_magic = 0x5d0520b8b7f2b168;
```

Comment:
Using `uint64_t` here rather than `char[8]` introduces an endianness dependency. Is this on purpose?

Reply:
It's not purposefully endian dependent, but purposefully uint64. Just looks nicer and the object is documented to be "implementation defined" anyway. But agree char should be better.

Reply:
fixed